### PR TITLE
Create config.yml to add new issue templates

### PR DESCRIPTION
**Pull request by cmgorton**
Adding a config.yml to give more options in our issues template. Will likely add some for documentation in the near future.

## Description
After discussing our issue templates in our forem/forem and forem/selfhost repo with Peter, we decided to add more templates to better direct people to the appropriate places for bug reports, features, and discussions. I will add a documentation link in the near future to help contributors create a better issue for documentation updates.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Feature Request
+    url: https://forem.dev/t/features
```
Comment (jdoss) [inline .github/ISSUE_TEMPLATE/config.yml:4]: Maybe talk about adding the selfhost tag instead of features?

**Review by jdoss — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (cmgorton) [reply .github/ISSUE_TEMPLATE/config.yml:4]: I think we would want to keep the features tag for the Feature requests but maybe mention adding the selfhost tag as well if it is a feature for selhost. I think that kind of language would be most useful in the actual feature template which I can modify on forem.dev.

Comment (jdoss) [reply .github/ISSUE_TEMPLATE/config.yml:4]: Sounds good to me.

**Review by codesandbax — COMMENTED** [review]
#33 #68 #69

**Pull request merged**